### PR TITLE
GitHub actions validation

### PR DESCRIPTION
**Pull request by dstenroejl**
AC from:
https://app.zenhub.com/workspaces/the-outlaws-6193fe815d79fc0011e741b1/issues/gh/energinet-datahub/the-outlaws/966

### CODE REVIEW

File: .github/workflows/xmlconversion-bundle-publish.yml [hunk 1]
```diff
@@ -47,33 +46,33 @@ jobs:
...
 
       - name: Pack XmlConverter.SimpleInjector project
-        uses: Energinet-DataHub/.github/.github/actions/nuget-project-pack@v9
+        uses: Energinet-DataHub/.github/.github/actions/nuget-project-pack@v10
         with:
-          PROJECT_PATH: './source/XmlConversion/source/XmlConverter.SimpleInjector/XmlConverter.SimpleInjector.csproj'
+          project_path: ./source/XmlConversion/source/XmlConverter.SimpleInjector/XmlConverter.SimpleInjector.csproj
 
       - name: Package content or build has changed
         id: changed-content
```
Comment (djorgensendk) [inline .github/workflows/xmlconversion-bundle-publish.yml:78]: tj-actions/changed-files is available in [v35](https://github.com/tj-actions/changed-files/releases/tag/v35)

File: README.md [hunk 2]
```diff
@@ -1,5 +1,3 @@
-[![codecov](https://codecov.io/gh/Energinet-DataHub/geh-core/branch/main/graph/badge.svg?token=CXGH54CZ85)](https://codecov.io/gh/Energinet-DataHub/geh-core)
-
```
Comment (djorgensendk) [inline README.md:2]: Should this badge be moved elsewhere -I'll admit I know not if it provides any value

File: documents/development.md [hunk 3]
```diff
@@ -205,4 +205,4 @@ When a new libray is added to the repository, tracking of code coverage must be
   - "source/<new-folder-name>/source/<test-folder-name>.Tests"
 ```
 
-* In the pipeline step using the action named `Energinet-DataHub/.github/.github/actions/dotnet-solution-build-and-test` add a argument named `CODE_COVERAGE_FLAGS`, and assign it the value given to the flag in `codecov.yml`.
\ No newline at end of file
+* In the pipeline step using the action named `Energinet-DataHub/.github/.github/actions/dotnet-solution-build-and-test` add a argument named `CODE_COVERAGE_FLAGS`, and assign it the value given to the flag in `codecov.yml`.
```
Comment (djorgensendk) [inline documents/development.md:208]: CODE_COVERAGE_FLAGS <-- lowercase?

Comment (dstenroejl) [reply documents/development.md:208]: Fixed

File: README.md (hunk 2 above)
Comment (dstenroejl) [reply README.md:2]: I don't believe it brings any value here, so I deleted it.

File: .github/workflows/xmlconversion-bundle-publish.yml (hunk 1 above)
Comment (dstenroejl) [reply .github/workflows/xmlconversion-bundle-publish.yml:78]: Fixed

**Pull request merged**